src/desktop/clone_system.desktop

```diff
@@ -0,0 +1,22 @@
+[Desktop Entry]
+Type=Application
+Categories=Settings;System;Qt;X-SuSE-YaST;X-SuSE-YaST-Misc;
+
+X-KDE-ModuleType=Library
+X-KDE-HasReadOnlyMode=true
+X-SuSE-YaST-Call=clone_system
+
+X-SuSE-YaST-Group=Misc
+X-SuSE-YaST-Argument=
+X-SuSE-YaST-RootOnly=true
+X-SuSE-YaST-AutoInst=
+X-SuSE-YaST-Geometry=
+X-SuSE-YaST-SortKey=
+X-SuSE-YaST-AutoInstResource=
+
+Icon=yast-autoyast
+Exec=/usr/bin/xdg-su -c '/sbin/yast2 clone_system'
+
+Name=Autoinstallation Cloning System
```

Comment:
Don't forget to update https://svn.opensuse.org/viewvc/opensuse-i18n/branches/trunk/lcn/50-pot/update-desktop-files-yast.pot?view=markup